src/OpenTelemetry.Exporter.Geneva/GenevaExporterHelperExtensions.cs

```diff
@@ -26,7 +28,7 @@ public static TracerProviderBuilder AddGenevaTraceExporter(this TracerProviderBu
     /// Adds <see cref="GenevaTraceExporter"/> to the <see cref="TracerProviderBuilder"/>.
     /// </summary>
     /// <param name="builder"><see cref="TracerProviderBuilder"/> builder to use.</param>
-    /// <param name="configure">Exporter configuration options.</param>
+    /// <param name="configure">Callback action for configuring <see cref="GenevaExporterOptions"/>.</param>
     /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
     public static TracerProviderBuilder AddGenevaTraceExporter(this TracerProviderBuilder builder, Action<GenevaExporterOptions> configure)
```

Comment:
The other method has nullable callback. Same for MetricsExporter.

```suggestion
    public static TracerProviderBuilder AddGenevaTraceExporter(this TracerProviderBuilder builder, Action<GenevaExporterOptions>? configure)
```

Reply:
Good catch. I've been going back and forth on this one myself 🤔 I decided this was the best shape for the API.

Short answer, it is consistent with what we did over here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs

Longer answer...

Let's say we have these overloads:

```csharp
public static MeterProviderBuilder AddMyExporter(this MeterProviderBuilder builder);
public static MeterProviderBuilder AddMyExporter(this MeterProviderBuilder builder, Action<MyOptions>? configure);
```

If user doesn't want to configure anything, why would they do this...

```csharp
builder.AddMyExporter(configure: null);
```

...instead of just...

```csharp
builder.AddMyExporter();
```

I decided to do `Action<MyOptions> configure` instead of `Action<MyOptions>? configure` because user passing `null` there for configuration delegate doesn't seem to be intentional/useful.

IMO this gives us a more logical progression:

```csharp
public static MeterProviderBuilder AddMyExporter(this MeterProviderBuilder builder);
public static MeterProviderBuilder AddMyExporter(this MeterProviderBuilder builder, Action<MyOptions> configure);
public static MeterProviderBuilder AddMyExporter(this MeterProviderBuilder builder, string? name, Action<MyOptions>? configure);
```

PS: There is no verification being added that the value actually is not `null`, it still allows `null` for backwards compatibility.

Reply:
Thanks! I am not against this. Just double-checking this difference.